ts/components/ProfileTabIcon.tsx

```diff
@@ -21,11 +22,13 @@ class ProfileTabIcon extends React.PureComponent<Props> {
     // when new experimental features will be avaible, pick this flag from props
     const isExperimentalFeaturesEnabled = false;
     return (
-      <IconFont
-        size={size}
-        color={color}
-        name={isExperimentalFeaturesEnabled ? "io-profilo-exp" : "io-profilo"}
-      />
+      <View accessibilityLabel={""}>
```

Comment:
why not `accessible={false}`?
If we have to force `accessibilityLabel={""}` could you leave a comment describing that decision (as done below)?

Reply:
is required on ios or he reads the "label" used to represent the font icon that we cannot change because is from third party. I've add some comments about that.